src/lib/utils/decoder.ts

```diff
@@ -1,9 +1,3 @@
-/**
- * @file ./lib is a great place to keep all your code.
- * You can then choose what to make available by default by
- * exporting your lib modules from the ./src/index.ts entrypoint.
- */
-
```

Comment:
Unrelated change, why is it here? be careful next time.

Reply:
Last 2 commits are unrelated,  I can push them after we merge this PR 👍